protocol/messenger_handler.go

```diff
@@ -1661,11 +1661,18 @@ func (m *Messenger) HandleCommunityRequestToJoinResponse(state *ReceivedMessageS
 	}
 
 	// check if it is outdated approved request to join
-	if requestToJoin.State != communities.RequestToJoinStatePending && requestToJoinResponseProto.Clock <= community.Clock() {
+	clockSeconds := requestToJoinResponseProto.Clock / 1000
+	isClockOutdated := clockSeconds < requestToJoin.Clock
```

Comment:
`requestToJoinResponseProto.Clock` is `community.Clock()` which is lamport timestamp, while `requestToJoin.Clock` is unix timestamp. I don't think comparing these two makes sense.

I am wondering why we need to determine whether the response is outdated, can't we simply ingore responses that falls under following condition `requestToJoin.State != communities.RequestToJoinStatePending`?